test/e2e/resize_nodes.go

```diff
@@ -373,16 +378,19 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	}
 
 	Logf("Waiting for node %s to be not ready", node.Name)
-	waitForNodeToBe(c, node.Name, false, 2*time.Minute)
+	if !waitForNodeToBe(c, node.Name, false, 2*time.Minute) {
```

Comment:
A few cleanups I'd have you do throughout (if you're willing):

0. Make the `2*time.Minute` a constant
0. Log how long we're going to wait with a `%v` every time we wait (this is incredibly helpful and not done a lot here)
0. Log why we're waiting each time we're doing it (why do we wait here and here and here for a node to be ready?)
0. Log which node wasn't ready and how long we waited when the check fails